### PR TITLE
feat: add Reunion to lat/longs

### PR DESCRIPTION
**Pull request by corneliusroemer**

### CODE REVIEW

**Comment by emmahodcroft** [comment]
Reunion is a territory of France - need to ensure ingest is correctly
assigning this. We've discussed different ways to address overseas
territories previously, but this is how we've handled them this far.

File: augur/data/lat_longs.tsv [hunk 1]
```diff
@@ -187,6 +187,7 @@ country	portugal	39.6945	-8.13057
 country	puerto_rico	18.2459121	-66.4164147
 country	puerto rico	18.2459121	-66.4164147
 country	qatar	25.27932	51.52245
+country reunion -21.1151 55.5364
```
Comment (jameshadfield) [inline augur/data/lat_longs.tsv:190]: Needs to change to tabs not spaces (at least, https://raw.githubusercontent.com/nextstrain/augur/c45ec7f828b3575f07a2caee67b833fc05a0666d/augur/data/lat_longs.tsv appears to be using spaces not tabs)

**Review by jameshadfield — COMMENTED** [review]
I can't remember our thinking about whether to include lat-longs for territories (and similar political classifications), but I think it's sensible for strains from Reunion to be shown in Reunion, in my opinion. I'll also note we have country lat-longs for American Samoa, British Virgin Islands etc.

File: augur/data/lat_longs.tsv (hunk 1 above)
Comment (corneliusroemer) [reply augur/data/lat_longs.tsv:190]: oops yes!
and I agree, not much point showing reunion sequences in France just because of politics...

**Comment by corneliusroemer** [comment]
Reunion is classified as a country by GISAID, I use the GISAID augur export directly, which is how I noticed

**Review by jameshadfield — APPROVED** [review]

**Comment by jameshadfield** [comment]
P.S. We have [specific GISAID annotation rules](https://raw.githubusercontent.com/nextstrain/ncov-ingest/master/source-data/gisaid_geoLocationRules.tsv) which change the country to France for Reunion sequences, so adding this lat-long won't change builds provisioned from that data.

**Pull request merged**